tests/sensitivities/test_create_design.py

```diff
@@ -203,3 +203,9 @@ def test_generate_full_mc(tmpdir):
     )
     assert not diskdefaults.empty
     assert len(diskdefaults.columns) == 2
+
+    # Make sure adding dependent discrete parameters works.
+    disk_depends = pd.read_excel(inputfile, sheet_name="depend1", engine="openpyxl")
+    df_merged = pd.merge(diskdesign, disk_depends, on="DATO", how="inner")
```

Comment:
"DATO" is not misspelled "DATE"? Or is it really called DATO in the file on disk maybe

Reply:
![image](https://github.com/user-attachments/assets/18a6a00f-223c-4f2b-b186-5bc160fd5143)

The `depend1` Sheet of `design_input_mc_with_correls` specifies `DATO`.